src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ConsentShutter.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentShutterModel
+@{
+    ViewData["Title"] = "Cannot connect family to service";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-fieldset__heading">
```

Comment:
The class should be `govuk-heading-l` and `govuk-fieldset__heading` should be removed.

Reply:
Changed